packages/opencensus-core/test/test-sampler.ts

```diff
@@ -100,4 +109,31 @@ describe('Sampler', () => {
          });
        });
   });
-});
\ No newline at end of file
+  describe('getNumberOfAnnotationEventsPerSpan', () => {
+    it('should return  12', () => {
```

Comment:
Could you make one of these test that values over the maximum get set to the maximum?

Reply:
@vigneshtdev any update here?

Reply:
It's been added already. https://github.com/census-instrumentation/opencensus-node/blob/3d8e965612d55a98c53e4947ac174af7e4db5c13/packages/opencensus-core/test/test-tracer.ts#L255